service/src/main/java/uk/gov/hmcts/reform/fpl/utils/BigDecimalHelper.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.fpl.utils;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.math.BigDecimal;
+import java.util.Optional;
+
+@Slf4j
+public class BigDecimalHelper {
+
+    private static final BigDecimal HUNDRED = BigDecimal.valueOf(100);
+
+    private BigDecimalHelper() {
+
+    }
+
+    public static String toCCDMoneyGBP(BigDecimal amount) {
+        if (amount == null) {
```

Comment:
```
return Optional.of(amount)
    .map(a -> a.multiply(HUNDRED))
    .map(BigDecimal::longValue)
    .map(String::valueOf)
    .orElse(StringUtils.EMPTY);
```

Reply:
merged the first 2 maps as well with a private method